proto/pstake/liquidstakeibc/v1beta1/liquidstakeibc.proto

```diff
@@ -216,7 +252,7 @@ message ValidatorUnbonding {
   string ibc_sequence_id = 6;
 }
 
-message KVUpdate {
+message KV {
```

Comment:
probably for the best if we do not use this? 
for better customisbility we can rather..
`Flags { LSM: bool, LSMConfig : {}..., Unstake: bool, UnstakeConfig: {AdminOnly:""}` , etc.. better to keep this all proto, rather than string encodings.
just as an example, we just need lsm bool for now

Reply:
Let's do it that way, wanted to get some flexibility with the KV, but there won't be that many flags after all